.bemlintrc.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
<img width="851" alt="image" src="https://user-images.githubusercontent.com/38065883/165377160-23813788-9a09-4ed8-bbe9-548ea82e2548.png">
There should another color

Reply:
<img width="839" alt="image" src="https://user-images.githubusercontent.com/38065883/165377272-c3740381-7be2-499e-9aea-c288668ae09b.png">
Only button should be scaled on hover, not entire card

https://user-images.githubusercontent.com/38065883/165377364-74b05b2d-62bd-4aec-821f-b874b0c6370d.mov

Reply:
<img width="921" alt="image" src="https://user-images.githubusercontent.com/38065883/165377462-df46b28c-b0f1-4fee-ac02-5c88d3755f14.png">
Export icons in svg out of figma, not png/jpg

Reply:
BTW @aliceblunt8 mentioned already that icon should become blue only on hover

Reply:
<img width="426" alt="image" src="https://user-images.githubusercontent.com/38065883/165377568-3f48e1b5-bc0e-4d13-bf25-29e964168fea.png">
There should be no standard border around this button

Reply:
<img width="694" alt="image" src="https://user-images.githubusercontent.com/38065883/165377662-dfff8309-4e88-4c2d-862a-0cf1d974198e.png">
There are still cards showing despite opened menu